src/styles/mocto-theme.scss

```diff
@@ -21,9 +21,12 @@ $mocto-disabled-light: #E9E9E9;
 $mocto-disabled: #C4C4C4;
 $mocto-disabled-dark: #7B7B7B;
 
+$mocto-box-height: 19px;
 $mocto-button-border-width: 2px;
-$mocto-link-border-width: 4px;
-$mocto-link-padding: 2px;
+$mocto-link-border-width: 4px; /* TODO: renaming needed so this naming can be used for actual links */
+$mocto-link-padding: 2px; /* TODO: renaming needed so this naming can be used for actual links */
```

Comment:
You use these tokens for the menu in the mobile view, maybe consider renaming them?

Reply:
I haven't specifically worked on the mobile view yet. 

Also, I mentioned this in another PR, but I no longer use mocto variables in the theme file. I have been able to just use spacing tokens in the class overrides file. I think doing that here would make sense

Reply:
Spacing tokens in the class overrides file completed! MOCTO variables are not used for spacing anymore. The other variables will be dealt with in #19 .